docs/eventing/migration/ping.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Migrating to the PingSource"
+weight: 20
+type: "docs"
+aliases:
+   - /docs/eventing/ping.md
+---
+
+The _deprecated_ `CronJobsource` should be converted to the newer `PingSource`.
+
+Imagine you have a `CronJobSource` running in your cluster that emits its events to a Knative Serving Service.
+The installation YAML would look like:
+
+```yaml
+apiVersion: sources.eventing.knative.dev/v1alpha1
+kind: CronJobSource
+metadata:
+  name: cronjob-source
+spec:
+  schedule: "* * * * *"
+  data: '{"message": "Hello world!"}'
+  sink:
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    name: event-display
+```
+
+If you now want to migration to the newer `PingSource`, a couple of steps are needed:
+
+* different GVK (see `apiVersion` and `kind` differences) 
+* the `PingSources` does use `jsonData` instead of `data`
```

Comment:
Instead:
"PingSource uses the `jsonData` attribute instead of the `data` attribute.